packages/amplication-client/src/context/appContext.tsx

```diff
@@ -31,7 +31,7 @@ export interface AppContextInterface {
   pendingChanges: PendingChangeItem[];
   commitRunning: boolean;
   pendingChangesIsError: boolean;
-  addEntity: (entityId: string) => void;
+  addEntity: (entityId?: string) => void;
```

Comment:
Why we changed that to optional?

Reply:
you can see above that they call addEntity when a resource is deleted. 
AddEntity and addBlock are actually just a trigger to refresh the list from the server. 
Maybe we can refactor the code and change the names of the functions to something like regreshPendingChanges() but I am OK with the current change at this phase. 

If someone thinks the refactor is needed, you can open a separate issue to be implemented in the future

Reply:
@yuval-hazaz I'm merging this pr and opening an issue for renaming the function name